Dockerfile

```diff
@@ -10,13 +10,16 @@ RUN echo "@operable https://storage.googleapis.com/operable-apk/" > /etc/apk/rep
 ARG MIX_ENV
 ENV MIX_ENV ${MIX_ENV:-dev}
 
-# Install dependencies that will be used at runtime. Build
-# time dependencies are installed _and_ removed during the
-# build stage layer.
+# Install runtime dependencies & nice-to-have packages
+RUN apk update -U && apk add bash ca-certificates curl git openssl postgresql-client
+
+# Install Erlang
 RUN apk update -U && \
-    apk add erlang erlang-crypto erlang-dev erlang-ssh erlang-ssl erlang-mnesia erlang-syntax-tools erlang-parsetools \
-            erlang-xmerl elixir \
-            bash ca-certificates curl git openssl postgresql-client
+    apk add `apk search erlang | grep -E "cos|eldap|gs|mibs|snmp|common-test|jinterface|megaco|diameter|odbc|observer|orber|test-server" -v | sed "s/-18.3.2-r0//g"`
```

Comment:
It's definitely ugly but short of `apk add`-ing all the Erlang things it's the only way I could think of to keep the image size down.